viewer/packages/cad-parsers/src/utilities/types.ts

```diff
@@ -2,9 +2,10 @@
  * Copyright 2021 Cognite AS
  */
 
-import * as THREE from 'three';
 import { SectorMetadata } from '../metadata/types';
 
+import * as THREE from 'three';
+
```

Comment:
```suggestion
import * as THREE from 'three';

import { SectorMetadata } from '../metadata/types';


```

Reply:
Wait, do we have a preferred order for these kinds of imports?? About time I found out, then :sweat_smile: